### ISSUE TITLE
native Apple M1 release?

### ISSUE THREAD

**Issue by imrefazekas** [zulip/zulip-desktop#1076]
Dear Team,

If the project is moved to Electron v11, it is possible to release an ARM version of the desktop app. To support Apple silicon platforms.
When could we expect such version?

Thanks!

**Comment by timabbott** [comment]
Thanks for the question!  We don't have a timeline yet, but we're looking into it.  Do you have an Apple M1 already?  If so, you can try building from source.

**Comment by michel-steuwer** [comment]
I have an M1 and can report that I successfully build it from source without any modifications resulting in an Apple silicon native process.

```
brew install node
npm install -g node-gyp

git clone https://github.com/zulip/zulip-desktop
cd zulip-desktop

npm install
npm start
```
just worked

**Comment by imrefazekas** [comment]
Yes it builds and it runs, thanks!

**Comment by timabbott** [comment]
Thanks for the confirmation!  I'm going to reopen this to track the work of including an M1 binary in our releases.

**Comment by andersk** [comment]
We just released Zulip Desktop 5.7.0, including an Apple ARM build: [Zulip-5.7.0-arm64.dmg](https://github.com/zulip/zulip-desktop/releases/download/v5.7.0/Zulip-5.7.0-arm64.dmg).
